src/common/helpers/proposalTitleAndDescription/approvedIdentifiersTable.txt

```diff
@@ -0,0 +1,244 @@
+|AAVEUSD| The price of AAVE in USD.| [UMIP-57](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-57.md)
```

Comment:
why do you need both this and the json file?

Reply:
The file downloaded from GitHub is raw markdown and needs to be parsed to JSON before we can use it. I'm leaving it here as a paper trail, but we could just as easily add a line at the end of the script to delete the file when we're done generating the JSON file.